README.md

```diff
@@ -27,4 +26,4 @@ Have a bug or feature request? Create one in the [PDS4 Issue Repo](https://githu
 
 # Developer Support
 
-[For more information](src/README.md)
+* [PDS Namespace Developer Documentation](https://pds-data-dictionaries.github.io/) 
```

Comment:
@acraugh do we want to point to some repo-specific developer docs here? We discussed `src/README.md` as the portal for the developers, which should probably point to these online docs you reference here, but can also include more.technical info about this specific repo. Or even just the directory structure and what it means, as noted in @mdrum's PR

Reply:
I don't think so, @jordanpadams, but I could probably be easily persuaded. My thinking was that the general information satisfies the curiosity of someone inspecting a repo to see what they might be getting into as a steward. Few repos would need technical documentation about the repo itself because they're all built on the same template. Geometry is the only one I can think of off-hand that might need special repo documentation because of its multiple input files. But anyone digging that deeply into the Geometry repo is going to see the README files in the subdirectories anyway. The directory structure is standardized.

I think I disagree with Mike's approach, but I'll bounce over there to comment on that.